cpp/src/umap/fuzzy_simpl_set/naive.h

```diff
@@ -168,14 +168,13 @@ __global__ void smooth_knn_dist_kernel(
   }
 }
 
+
 /**
  * Construct the membership strength data for the 1-skeleton of each local
  * fuzzy simplicial set -- this is formed as a sparse matrix (COO) where each
  * row is a local fuzzy simplicial set, with a membership strength for the
  * 1-simplex to each other data point.
  *
- * TODO: Optimize for coalesced reads (use col-major inputs).
```

Comment:
Removing this TODO. Now that this kernel is element-wise instead of row-wise, it will automatically reap the benefits of coalescing.